### DISCUSSION TITLE
Building heights

### DISCUSSION THREAD

**Discussion by fredxpel** [OvertureMaps/data#98] [Discussion]
Does this database provide accurate building heights?

**Comment by jenningsanderson** [comment]
The buildings theme contains building heights from various sources, each will have variations in accuracy. The `sources` column records the source for each attribute.

The current dataset has heights from the following sources: 
- OpenStreetMap (Community contributed height info)
- Esri Community Maps (Community contributed height info)
- USGS Lidar (Extracted from Lidar)
- Microsoft ML Buildings Dataset (AI-derived heights)
